js/script.js

```diff
@@ -100,6 +100,9 @@ $(function(){
 						self.handleActivitiesCallback([]);
 						self.saveHeaders(xhr.getAllResponseHeaders());
 						return;
+					} else if (status === 'nocontent'){
```

Comment:
shouldn't happen with 20 anymore, as you can not disable all activity settings anymore, but sure.